week1/project/ecommerce/src/App.css

```
@@ -0,0 +1,76 @@
+
+.products {
+    display: flex;
+    flex-wrap: wrap;
+    list-style: none;
+    justify-content: center;
+}
+
+.product {
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+    width: 600px;
+    margin: 20px;
+    word-wrap: break-word;
+}
+
+.product-img{
+    margin-bottom: auto;
+    margin-top: auto;
```

You can combine this is one line 😄 

```suggestion
    margin: auto 0;
```

---

Ooops, I'll fix it